src/DynamoCore/Graph/Workspaces/PackageDependencyInfo.cs

```diff
@@ -6,7 +6,7 @@ namespace Dynamo.Graph.Workspaces
     /// <summary>
     /// Class containing info about a package
     /// </summary>
-    internal class PackageInfo
+    public class PackageInfo
```

Comment:
Good, we still end up making it public. Would you check if there is any internal available code we need to remove?

Reply:
will do.